server/index.js

```diff
@@ -0,0 +1,27 @@
+var express = require('express');
+
+//middleware
+var morgan = require('morgan');
+var parser = require('body-parser');
+const cors = require('cors');
+
+// run express server
+var server = express();
+
+// enable CORS
+server.use(cors({
+  origin: '*',
+}));
+
+// logging & parsing
+server.use(morgan('dev'));
+server.use(parser.urlencoded({ extended: true }));
+server.use(parser.json());
+
+// serve client files
+server.use('/', express.static(__dirname + './../client/dist'));
```

Comment:
I think your proxy server will only serve up index.html at the root "/" path, it doesn't seem like it will recognize other paths like "/1/200"--so same issue we encountered before with our other services.

Did you try running your proxy yet?

Reply:
Good catch Hugo!  Added line 23 to serve up static files on Shop page.